### PR TITLE
LPS-195619 Convert old AUI in staging-version using JS

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
Hey, @liferay-staging!

I’v been working in some task removing AUI from the portal to use JS/TS instead and I made some changes for the staging_version.js.
Would you mind to take a look at it an share your thoughts about the changes? Thanks!

## Motivation

Replace old AUI files and try to convert it to JS/TS

[Link to ticket](https://liferay.atlassian.net/browse/LPS-195619)

## Test Scenario 1

1. Activate staging (with page versioning)
2. Create a new Widget Page.
3. Check the behaviour for all kebab menu actions (Page variations, Site variations, History, Redo and Undo).
4. Check the behaviour for the toggle of “Ready for Publish Process”    

https://github.com/liferay-echo/liferay-portal/assets/93203423/5874c316-5494-4bd2-bc4f-9e26cd3a90dd

5. Go to Workflow. Look for “page” and activate Page Revision.
6. Navigate again to the widget page and check the behaviour for the button of “Submit for Workflow”

https://github.com/liferay-echo/liferay-portal/assets/93203423/eca6314f-4ed3-4dec-bdef-438df1b05dff

### CODE REVIEW

**Comment by zsigmondrab** [comment]
ci:test:staging

**Comment by gaborlovasdotcom** [comment]
ci:test:bundle

File: modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp [hunk 1]
```diff
@@ -149,144 +166,32 @@ else {
...
-				<aui:icon cssClass="<%= StringPool.BLANK %>" image="ellipsis-v" markupView="lexicon" />
-
-				<span class="sr-only">
-					<liferay-ui:message key="options" />
-				</span>
-			</button>
-
-			<ul class="dropdown-menu dropdown-menu-right" role="menu">
-				<li>
-					<a class="dropdown-item" href="javascript:void(0);" id="manageLayoutSetRevisions" onclick="<portlet:namespace />openSitePagesVariationsDialog();">
```
Comment (gaborlovasdotcom) [inline modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp:164]: Hi @BarbaraCabrera , could you please keep this IDs, as they are reference points for Functional tests / Poshi tests?

File: modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp [hunk 2]
```diff
@@ -149,144 +166,32 @@ else {
...
+	</portlet:renderURL>
 
-				<c:if test="<%= !layoutRevision.isIncomplete() && !layout.isTypeContent() %>">
-					<li>
-						<a class="dropdown-item" href="javascript:void(0);" id="manageLayoutRevisions" onclick="<portlet:namespace />openPageVariationsDialog();">
-							<liferay-ui:message key="page-variations" />
-						</a>
-					</li>
-					<li>
-						<a class="dropdown-item" href="javascript:Liferay.fire('<portlet:namespace />viewHistory', {layoutRevisionId: '<%= layoutRevision.getLayoutRevisionId() %>', layoutSetBranchId: '<%= layoutRevision.getLayoutSetBranchId() %>'}); void(0);" id="viewHistoryLink">
```
Comment (gaborlovasdotcom) [inline modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp:176]: Hi @BarbaraCabrera , could you please keep this IDs, as they are reference points for Functional tests / Poshi tests?

**Comment by BarbaraCabrera** [comment]
Hey, @gaborlovasdotcom ! 😄 

I was taking a look at your messages and I want to share my thoughts about adding the ids.

The main reason why I think we don't have to use this ids it’s that we should avoid the use of specific code just for use it in the test, also I think the ideal is to test it as a user would, looking for related elements in the interface and not through elements that are not available for the user to see or use.
Also, I think this kind of testing is already done in other places on the portal and it should be possible to do it without added elements for testing purposes only.

If this were not possible, we should review how clay elements are generated and how Poshi uses them to fix this kind of problems.

How would it be for you if we contributed some changes to the Poshi tests to help with this situation?

**Comment by BarbaraCabrera** [comment]
I'm closing this pr and I will open another one it with the new changes, thanks!

**Pull request closed without merging**